libc/startup/linux/aarch64/start.cpp

```diff
@@ -184,7 +184,7 @@ __attribute__((noinline)) static void do_start() {
     app.tls.align = phdr->p_align;
   }
 
-  LIBC_NAMESPACE::TLSDescriptor tls;
+  static LIBC_NAMESPACE::TLSDescriptor tls;
```

Comment:
is changing this to static necessary?

Reply:
Unfortunately yes, the lambda expression to the `atexit` routine must be compatible with `void (*)()` hence capturing nothing.